src/Shaders/ShadersInclude/morphTargetsVertex.fx

```diff
@@ -3,5 +3,10 @@
 	
 	#ifdef MORPHTARGETS_NORMAL
 	normalUpdated += (normal{X} - normal) * morphTargetInfluences[{X}];
-	#endif	
+	#endif
+
+	#ifdef MORPHTARGETS_TANGENT
+	tangentTemp = tangentUpdated.xyz + ((tangent{X} - tangent.xyz) * morphTargetInfluences[{X}]);

```

Comment:
This seems suspicious.
why not tangentUpdated += (tangent{X} - normal) * morphTargetInfluences[{X}];